docs/platform-packet.md

```diff
@@ -62,9 +68,10 @@ PACKET_API_KEY=<API key> PACKET_PROJECT_ID=<Project ID> \
 linuxkit run packet -serve :8080 -base-url <ngrok url> packet
 ```
 
-To boot a `arm64` image for Type 2a machine (`-machine
-baremetal_2a`) you currently need build using `linuxkit build packet.yml packet.arm64.yml` and then un-compress both the kernel and
-the initrd before booting, e.g:
+To boot a `arm64` image for Type 2a machine (`-machine baremetal_2a`)
+you currently need build using `linuxkit build packet.yml
```

Comment:
"...currently need to build using..." (extra "to")

Reply:
fixed